cmd/bootstrapper/configure_cni.go

```diff
@@ -66,7 +66,8 @@ func runConfigureCNICmd(cmd *cobra.Command, args []string) {
 		log.Error(err, "could not configure CNI")
 		os.Exit(1)
 	}
-	log.Info("CNI configuration completed successfully")
+	// Send success message to StdOut for WSU to ascertain that CNI configuration was successful
```

Comment:
nit: mentioning WSU is strange to me since we are phasing it out... but really doesn't matter much to me

Reply:
That's a good point - I only mentioned it because only WSU needs these success messages to move forward, and consequently these success messages could be logged instead once we phase out WSU